src/extensions/default/InlineBorderRadiusEditor/BorderRadiusEditor.js

```diff
@@ -0,0 +1,818 @@
+
+define(function(require, exports, module) {
+    "use strict";
+
+    var KeyEvent           = brackets.getModule("utils/KeyEvent"),
+        PreferencesManager = brackets.getModule("preferences/PreferencesManager"),
+        StringUtils        = brackets.getModule("utils/StringUtils"),
+        Strings            = brackets.getModule("strings"),
+        Mustache           = brackets.getModule("thirdparty/mustache/mustache"),
+        BorderRadiusUtils   = brackets.getModule("utils/BorderRadiusUtils");
+
+       // tinycolor          = require("thirdparty/tinycolor-min")
+
+    /** Mustache template that forms the bare DOM structure of the UI */
+    var BorderRadiusTemplate = require("text!BorderRadiusEditorTemplate.html");
+    var DEFAULT_BORDER_RADIUS_VALUE = 0;
+    /**
+     * Box shadow editor control; may be used standalone or within an InlineBoxShadowEditor inline widget.
+     * @param {!jQuery} $parent  DOM node into which to append the root of the box-shadow editor UI
+     * @param {!{horizontalOffset: string, verticalOffset: string, blurRadius: string, spreadRadius: string, color: string}} values  Initial set of box-shadow values.
+     * @param {!function(string)} callback  Called whenever values change
+     */
+    function getIndividualValues(values){
+        var flag = true;
+        String.prototype.replaceAll = function(str1, str2, ignore)
```

Comment:
Let's not do this.  Modifying the `prototype` of a built-in is a bad idea, since it will pollute global scope.  Instead, please turn this into a regular function you call as needed.  Also, always put `{` on the same line as the function.